### PR TITLE
fix: Update the Chinese translation in lr-scheduler.md

### PR DESCRIPTION
**Pull request by Nietism**
+ Correct a translation error. In previous Chinese edition of ``lr-scheduler.md``, "Factor Scheduler" and "Multi Factor Scheduler" are translated into the same word "多因子调度器" and it might confuse readers before they read the original discourse.

+ Rewrite several sentences because they were incorrectly or improperly statemented due to machine translation.

### CODE REVIEW

File: chapter_optimization/lr-scheduler.md [hunk 1]
```diff
@@ -427,8 +430,8 @@ scheduler = CosineScheduler(max_update=20, base_lr=0.3, final_lr=0.01)
 d2l.plot(d2l.arange(num_epochs), [scheduler(t) for t in range(num_epochs)])
 ```
 
-在计算机视觉中，这个调度可以引出改进的结果。
-但请注意，如下所示，这种改进并不能保证成立。
+在计算机视觉的背景下，这个调度方式可能产生改进的结果。
+但请注意，如下所示，并不能保证这种改进一定成立。
```
Comment (goldmermaid) [inline chapter_optimization/lr-scheduler.md:434]:
```suggestion
但请注意，如下所示，这种改进并不一定成立。
```

**Review by goldmermaid — APPROVED** [review]
Great work @Nietism ! Could you make the minor changes?

**Comment by Nietism** [comment]
> Great work @Nietism ! Could you make the minor changes?

Thanks for you suggestion! The previous translating is not good indeed.

File: chapter_optimization/lr-scheduler.md (hunk 1 above)
Comment (Nietism) [reply chapter_optimization/lr-scheduler.md]: Have finished commit merging and been built yet.
